takeshape/penny
# Handle inventory and availability correctly

### Test Plan (Steps to test):

1. Pull this branch
1. `npm run dev`
2. Go to a product page
1. On the DSP store in Shopify, find that product.
3. Toggle the channel availability of that product and reload the page. It should now be not found.
4. Toggle the channel availability back. Reload the page. Should be there.
5. Toggle the status in Shopify, reload, etc, should see similar behavior.
6. Go to a product with multiple variants. Ensure one variant is out of stock. Load that product on your dev site. You should not be able to select the out of stock variant.
7. Toggle off the inventory tracking for that variant. It should now be selectable.
